tests/testsuite/build_script.rs

```diff
@@ -787,7 +787,7 @@ fn custom_build_script_wrong_rustc_flags() {
         .file("src/main.rs", "fn main() {}")
         .file(
             "build.rs",
-            r#"fn main() { println!("cargo:rustc-flags=-aaa -bbb"); }"#,
+            r#"fn main() { println!("cargo::rustc-flags=-aaa -bbb"); }"#,
```

Comment:
Since some things are going to get revisited due to `cargo:KEY=VALUE`, I'm holding off on digging deeper into the tests but some quick guidance

- We should make sure we have tests verifying the old syntax so we don't break it
- When a test isn't specifically covering the old syntax, then yes, we should be using the new syntax

Reply:
You can checkout https://github.com/rust-lang/cargo/pull/12201/commits/4ea02bbed3c7051d471990ebb7395e8053adf3f0.